en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,100 @@
 # PX4 Continuous Integration
 
-PX4 builds and testing are performed using GitHub actions and a Jenkins server.
\ No newline at end of file
+PX4 uses continuous integration (CI) to build and run tests against every pull request (PR) and release, and to generate build artifacts such as firmware, documentation updates, and various metadata files needed by QGroundControl.
+
+The integration system uses GitHub actions and a Jenkins server.
+The configuration files for these systems can be found in the [PX4-Autopilot](https://github.com/PX4/PX4-Autopilot) repo main branch:
+- [Jenkinsfile](https://github.com/PX4/PX4-Autopilot/blob/main/Jenkinsfile)
+- [.github/workflows/](https://github.com/PX4/PX4-Autopilot/tree/main/.github/workflows).
+
+The Jenkins build status can be viewed in [ci.px4.io:8080/job/PX4/](http://ci.px4.io:8080/job/PX4/)
+
+This document outlines how the different parts of the CI system are configured and connected.
+
+## Pull Requests
+
+Automated CI checks are run for every [Pull-Request (PR)](https://github.com/PX4/PX4-Autopilot/pulls).
+
+Most of the test are run using the Github workflows in [.github/workflows](https://github.com/PX4/PX4-Autopilot/tree/main/.github/workflows):
+
+- `compile_nuttx.yml`:
+  Builds NuttX flight controller board targets.
+  It can detect Flash overflows and code syntax errors,
+- `sitl_tests.yml`: 
+  Executes [MAVSDK SITL Tests](./integration_testing_mavsdk.md).
+  Detect system-level bugs in the code (e.g. VTOL doesn't return to home)
```

Comment:
IMO worth expanding a bit here: It runs missions, offboard, failsafes, follow me, etc. The tests are under `test/mavsdk_tests`. They also upload the logs to flight review.